chatkit/src/main/java/com/pusher/chatkit/ChatManager.kt

```diff
@@ -146,12 +125,15 @@ class ChatManager @JvmOverloads constructor(
             locator = instanceLocator,
             serviceName = serviceName,
             serviceVersion = serviceVersion,
-            logger = logger,
-            baseClient = baseClient,
-            connectivityHelper = connectivityHelper,
-            scheduler = scheduler,
-            mainThreadScheduler = mainScheduler,
-            mediatypeResolver = mediaTypeResolver
+            dependencies = AndroidDependencies(
+                context,
+                SdkInfo(
+                    product = "Chatkit",
+                    sdkVersion = BuildConfig.VERSION_NAME,
+                    platform = "Android",
+                    language = "Kotlin/Java"
```

Comment:
Reaching a bit I expect, but is it possible to have this set to one or the other depending on some environment value? I guess it's gonna be like Swift / ObjC where you can have a mixed source app and so it's impossible to tell.

Reply:
Yeah, it's not straightforward to check, especially since the library itself brings Kotlin with it. I guess we could cover both cases by using `jvm` (even if it's ART, hehe)